### ISSUE TITLE
L'affichage du point de la carte sur le frontend est décalé en haut à droite au lieu d'être au milieu

### ISSUE THREAD

**Issue by nfavrod** [Skrattoune/OpenRepairPlatform#7]
https://demo.repaircafes.ch/location/1/pole-sud/
<img width="986" alt="Capture d’écran 2022-03-11 à 00 40 41" src="https://user-images.githubusercontent.com/1118004/157773861-2f46d4ed-d5dd-4e25-bf8d-dacbe558d2a5.png">

**Comment by nfavrod** [comment]
Ca n'aurait pas été résolu avec l'issue #25 ?

**Comment by Skrattoune** [comment]
non, ça n'a rien changé: https://dev-openrepairplateform.repaircafes.ch/location/1/pole-sud/
